### PR TITLE
Rename the Agent BARX and DEVX teams

### PR DESCRIPTION
**Pull request by FlorentClarret**
### What does this PR do?

Rename the Agent BARX and DEVX teams

### Motivation

Reorg

### CODE REVIEW

File: tasks/libs/pipeline/github_slack_map.yaml [hunk 1]
```diff
@@ -37,7 +37,7 @@
 '@datadog/agent-cspm': '#k9-cspm-ops'
 '@datadog/telemetry-and-analytics': '#instrumentation-telemetry'
 '@datadog/asm-go': '#k9-asm-library-go'
-'@datadog/agent-build-and-releases': '#agent-build-and-releases'
-'@datadog/agent-ci-experience': '#agent-developer-experience'
-'@datadog/agent-developer-tools': '#agent-developer-experience'
+'@datadog/agent-delivery': '#agent-delivery'
```
Comment (FlorentClarret) [inline tasks/libs/pipeline/github_slack_map.yaml:40]:
```suggestion
'@datadog/agent-delivery': '#agent-build-and-releases'
```

**Review by pgimalac — APPROVED** [review]

**Review by chouquette — APPROVED** [review]

File: tasks/libs/pipeline/github_slack_map.yaml [hunk 2]
```diff
@@ -37,7 +37,7 @@
 '@datadog/agent-cspm': '#k9-cspm-ops'
 '@datadog/telemetry-and-analytics': '#instrumentation-telemetry'
 '@datadog/asm-go': '#k9-asm-library-go'
-'@datadog/agent-build-and-releases': '#agent-build-and-releases'
-'@datadog/agent-ci-experience': '#agent-developer-experience'
-'@datadog/agent-developer-tools': '#agent-developer-experience'
+'@datadog/agent-delivery': '#agent-build-and-releases'
+'@datadog/agent-devx-infra': '#agent-developer-experience'
+'@datadog/agent-devx-loops': '#agent-developer-experience'
```
Comment (KevinFairise2) [inline tasks/libs/pipeline/github_slack_map.yaml:42]:
```suggestion
'@datadog/agent-devx-infra': '#agent-devx-help'
'@datadog/agent-devx-loops': '#agent-devx-help'
```

File: tasks/unit-tests/libs/pipeline/failure_summary_tests.py [hunk 3]
```diff
@@ -287,14 +287,14 @@ def test_send_summary_messages(self, mock_slack: MagicMock = None):
                     )
                     for _ in range(10)
                 ],
-                # agent-ci-experience, agent-build-and-releases
+                # agent-devx-infra, agent-build-and-releases
```
Comment (KevinFairise2) [inline tasks/unit-tests/libs/pipeline/failure_summary_tests.py:290]:
```suggestion
                # agent-devx-infra, agent-delivery
```

File: tasks/unit-tests/libs/pipeline/failure_summary_tests.py [hunk 4]
```diff
@@ -287,14 +287,14 @@ def test_send_summary_messages(self, mock_slack: MagicMock = None):
...
                 *[
                     ProjectPipelineJob(
                         manager=MagicMock(), attrs={'name': 'tests_release', 'status': 'failed', 'allow_failure': False}
                     )
                     for _ in range(5)
                 ],
-                # agent-ci-experience, agent-build-and-releases
+                # agent-devx-infra, agent-build-and-releases
```
Comment (KevinFairise2) [inline tasks/unit-tests/libs/pipeline/failure_summary_tests.py:297]:
```suggestion
                # agent-devx-infra, agent-delivery
```

File: tasks/unit-tests/testdata/github_slack_map.yaml [hunk 5]
```diff
@@ -37,7 +37,7 @@
 '@datadog/agent-cspm': '#k9-cspm-ops'
 '@datadog/telemetry-and-analytics': '#instrumentation-telemetry'
 '@datadog/asm-go': '#k9-asm-library-go'
-'@datadog/agent-build-and-releases': '#agent-build-and-releases'
-'@datadog/agent-ci-experience': '#agent-developer-experience'
-'@datadog/agent-developer-tools': '#agent-developer-experience'
+'@datadog/agent-delivery': '#agent-build-and-releases'
```
Comment (KevinFairise2) [inline tasks/unit-tests/testdata/github_slack_map.yaml:40]:
```suggestion
'@datadog/agent-delivery': '#agent-delivery-help'
```

File: tasks/unit-tests/testdata/github_slack_map.yaml [hunk 6]
```diff
@@ -37,7 +37,7 @@
...
+'@datadog/agent-devx-infra': '#agent-developer-experience'
+'@datadog/agent-devx-loops': '#agent-developer-experience'
```
Comment (KevinFairise2) [inline tasks/unit-tests/testdata/github_slack_map.yaml:42]:
```suggestion
'@datadog/agent-devx-infra': '#agent-devx-help'
'@datadog/agent-devx-loops': '#agent-devx-help'
```

File: tasks/libs/pipeline/github_slack_map.yaml (hunk 2 above)
Comment (FlorentClarret) [reply tasks/libs/pipeline/github_slack_map.yaml:42]: done in another PR https://github.com/DataDog/datadog-agent/pull/26965

File: tasks/unit-tests/libs/pipeline/failure_summary_tests.py (hunk 3 above)
Comment (FlorentClarret) [reply tasks/unit-tests/libs/pipeline/failure_summary_tests.py:290]: done in another PR https://github.com/DataDog/datadog-agent/pull/26965

File: tasks/unit-tests/libs/pipeline/failure_summary_tests.py (hunk 4 above)
Comment (FlorentClarret) [reply tasks/unit-tests/libs/pipeline/failure_summary_tests.py:297]: done in another PR https://github.com/DataDog/datadog-agent/pull/26965

File: tasks/unit-tests/testdata/github_slack_map.yaml (hunk 5 above)
Comment (FlorentClarret) [reply tasks/unit-tests/testdata/github_slack_map.yaml:40]: done in another PR https://github.com/DataDog/datadog-agent/pull/26965

File: tasks/unit-tests/testdata/github_slack_map.yaml (hunk 6 above)
Comment (FlorentClarret) [reply tasks/unit-tests/testdata/github_slack_map.yaml:42]: done in another PR https://github.com/DataDog/datadog-agent/pull/26965

**Review by KevinFairise2 — APPROVED** [review]

File: tasks/unit-tests/libs/pipeline/failure_summary_tests.py (hunk 3 above)
Comment (chouetz) [inline tasks/unit-tests/libs/pipeline/failure_summary_tests.py:290]:
```suggestion
                # agent-devx-infra, agent-delivery
```

File: tasks/unit-tests/pipeline_lib_tests.py [hunk 7]
```diff
@@ -12,11 +12,11 @@ class TestLoadAndValidate(unittest.TestCase):
     def test_files_loaded_correctly(self):
         # Assert that a couple of expected entries are there, including one that uses DEFAULT_JIRA_PROJECT
         self.assertEqual(notifications.GITHUB_JIRA_MAP['@datadog/agent-all'], "AGNTR")
-        self.assertEqual(notifications.GITHUB_JIRA_MAP['@datadog/agent-ci-experience'], "ACIX")
+        self.assertEqual(notifications.GITHUB_JIRA_MAP['@datadog/agent-devx-infra'], "ACIX")
 
         # Assert that a couple of expected entries are there, including one that uses DEFAULT_SLACK_PROJECT
         self.assertEqual(notifications.GITHUB_SLACK_MAP['@datadog/agent-all'], "#datadog-agent-pipelines")
-        self.assertEqual(notifications.GITHUB_SLACK_MAP['@datadog/agent-ci-experience'], "#agent-developer-experience")
+        self.assertEqual(notifications.GITHUB_SLACK_MAP['@datadog/agent-devx-infra'], "#agent-developer-experience")
```
Comment (chouetz) [inline tasks/unit-tests/pipeline_lib_tests.py:19]:
```suggestion
        self.assertEqual(notifications.GITHUB_SLACK_MAP['@datadog/agent-devx-infra'], "#agent-devx-help")
```

File: tasks/unit-tests/testdata/github_slack_map.yaml (hunk 6 above)
Comment (chouetz) [inline tasks/unit-tests/testdata/github_slack_map.yaml:42]:
```suggestion
'@datadog/agent-delivery': '#agent-delivery-help'
'@datadog/agent-devx-infra': '#agent-devx-help'
'@datadog/agent-devx-loops': '#agent-devx-help'
```

File: tasks/libs/pipeline/github_slack_map.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (FlorentClarret) [reply tasks/libs/pipeline/github_slack_map.yaml:42]: https://github.com/DataDog/datadog-agent/pull/26956#discussion_r1650536514

File: tasks/unit-tests/junit_tests.py [hunk 8]
```diff
@@ -91,7 +91,7 @@ def test_default(self, mock_gitlab):
         mock_instance = MagicMock()
         mock_instance.pipelines.get.return_value = MagicMock()
         mock_gitlab.return_value = mock_instance
-        tags = junit.set_tags("agent-ci-experience", "base", "", {}, "")
+        tags = junit.set_tags("agent-devx-infra", "base", "", {}, "")
         self.assertEqual(len(tags), 14)
         self.assertIn("slack_channel:agent-developer-experience", tags)
```
[the remark this answers is not in the document]
Comment (FlorentClarret) [reply tasks/unit-tests/junit_tests.py:96]: https://github.com/DataDog/datadog-agent/pull/26956#discussion_r1650536514

File: tasks/unit-tests/libs/pipeline/failure_summary_tests.py (hunk 3 above)
Comment (FlorentClarret) [reply tasks/unit-tests/libs/pipeline/failure_summary_tests.py:290]: https://github.com/DataDog/datadog-agent/pull/26956#discussion_r1650536514

File: tasks/unit-tests/pipeline_lib_tests.py (hunk 7 above)
Comment (FlorentClarret) [reply tasks/unit-tests/pipeline_lib_tests.py:19]: https://github.com/DataDog/datadog-agent/pull/26956#discussion_r1650536514

File: tasks/unit-tests/testdata/github_slack_map.yaml (hunk 6 above)
Comment (FlorentClarret) [reply tasks/unit-tests/testdata/github_slack_map.yaml:42]: https://github.com/DataDog/datadog-agent/pull/26956#discussion_r1650536514

**Review by pducolin — APPROVED** [review]

File: tasks/libs/pipeline/github_slack_map.yaml [hunk 9]
```diff
@@ -37,7 +37,7 @@
 '@datadog/agent-cspm': '#k9-cspm-ops'
 '@datadog/telemetry-and-analytics': '#instrumentation-telemetry'
 '@datadog/asm-go': '#k9-asm-library-go'
-'@datadog/agent-build-and-releases': '#agent-delivery-ops'
-'@datadog/agent-ci-experience': '#agent-devx-ops'
-'@datadog/agent-developer-tools': '#agent-devx-ops'
+'@datadog/agent-delivery': '#agent-delivery-ops'
+'@datadog/agent-devx-infra': '#agent-delivery-ops'
+'@datadog/agent-devx-loops': '#agent-delivery-ops'
```
Comment (FlorentClarret) [inline tasks/libs/pipeline/github_slack_map.yaml:42]:
```suggestion
'@datadog/agent-delivery': '#agent-devx-ops'
'@datadog/agent-devx-infra': '#agent-devx-ops'
'@datadog/agent-devx-loops': '#agent-devx-ops'
```

**Review by amenasria — APPROVED** [review]
LGTM

**Pull request merged**